File: 프로그래머스/Level2/최솟값 만들기.py
# https://programmers.co.kr/learn/courses/30/lessons/12941

def solution(A, B):
    A.sort()
    B.sort()
    answer = 0
    j = len(B) - 1
    for i in range(len(A)):
        answer += A[i] * B[j]
        j -= 1
    return answer

File: 프로그래머스/Level2/최솟값 만들기.js
// https://programmers.co.kr/learn/courses/30/lessons/12941

const solution = (A, B) => {
	A.sort((a, b) => a - b);
	B.sort((a, b) => b - a);
	return A.reduce((prev, cur, idx) => prev + cur * B[idx], 0);
};
